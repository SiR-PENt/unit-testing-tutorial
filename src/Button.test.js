import React from 'react';
import renderer from 'react-test-renderer';
import Button from './Button';

test('Button should match its snapshot', () => {
  const button = renderer.create(<Button />).toJSON();
  expect(button).toMatchSnapshot();
});
