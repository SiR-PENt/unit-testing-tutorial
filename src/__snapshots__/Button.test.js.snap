// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should match its snapshot 1`] = `
<button>
  Click Me
</button>
`;
